### DISCUSSION TITLE
Access multi-select optionGroupLabel name.

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#504] [PrimeVue]
How can I access to multi-select optionGroupLabel name to check if group name is equal "color" for example?

 <PrimeSelect
                v-model="selectedFilters"
                filter
                :options="carsFilters"
                optionGroupLabel="name"
                optionGroupChildren="values"
                placeholder="Filters"
                class="w-full md:w-15rem mt-8 col-lg-2 col-md-6 col-sm-12"
                panelClass="z-1"
             >

        <template #option="slotProps">
            <div class="flex align-items-center">
                <span class="mr-2" style="width: 15px; height: 15px; background-color: blue; border: 1px solid black; border-radius: 50%;"></span>
                <div>{{ slotProps.option.name }}</div>
            </div>
        </template>
    </PrimeSelect>

**Comment by tugcekucukoglu** [comment]
Could you share a reproducer link?

**Reply by Uzeyir9695** [reply]
Sorry, I didn't get you what you mean by saying reproducer link.
